plugins/ua_network_pubsub_udp.h

```diff
@@ -0,0 +1,34 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information.
+ *
+ *    Copyright 2017-2018 (c) Fraunhofer IOSB (Author: Andreas Ebner)
+ */
+
+#ifndef UA_NETWORK_UDPMC_H_
+#define UA_NETWORK_UDPMC_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef _WIN32
+#include <sys/socket.h>
+#endif
+#include "ua_plugin_pubsub.h"
+
+//UDP multicast network layer specific internal data
+typedef struct {
+    int ai_family;                        //Protocol family for socket.  IPv4/IPv6
+    struct sockaddr_storage *ai_addr;     //https://msdn.microsoft.com/de-de/library/windows/desktop/ms740496(v=vs.85).aspx
```

Comment:
The plugins are exchangeable so that we allow different implementations.
Here we require a specific networking API in the plugin header already.
Can we move `UA_PubSubChannelDataUDPMC` inside the plugin so that the interface is agnostic?

Reply:
Let me rephrase the comment:
This is **inside** the plugin. So we can actually use system-specific API.
But since the structure is used only inside ` plugins/ua_network_pubsub_udp.c`, let's move the definition there.

Just to separate concerns and keep the API surface small.